packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/review/eventgrid.api.md

```diff
@@ -50,7 +50,9 @@ export class EventGridClient {
     constructor(endpointParam: string, credential: KeyCredential, options?: EventGridClientOptions);
     acknowledgeCloudEvents(topicName: string, eventSubscriptionName: string, lockTokens: AcknowledgeOptions, options?: AcknowledgeCloudEventsOptionalParams): Promise<AcknowledgeResult>;
     readonly pipeline: Pipeline;
-    publishCloudEvent(topicName: string, event: CloudEvent, options?: PublishCloudEventOptionalParams): Promise<PublishResult>;
+    publishCloudEvent(topicName: string, event: {
+        event: CloudEvent;
+    }, options?: PublishCloudEventOptionalParams): Promise<PublishResult>;
```

Comment:
this is expected, as the typespec is defined to have an anonymous body parameter for it.
```
  @post op PublishCloudEvent is Azure.Core.RpcOperation<{
    @doc("content type")
    @header("content-type")
    contentType: "application/cloudevents+json; charset=utf-8";

    @doc("Topic Name.")
    @path
    topicName: string;

    @doc("Single Cloud Event being published.")
    @body 
    event: {event: CloudEvent};
  }, { @body _: PublishResult; }>;
```